benchexec/tools/btor2c.py

```diff
@@ -12,25 +12,28 @@
 
 class Tool(benchexec.tools.template.BaseTool2):
     """
-    Tool info for BTOR2C: A Converter from BTOR2 models to C programs
+    Tool info for Btor2C: A translator from Btor2 circuits to C programs
     """
 
     def executable(self, tool_locator):
-        return tool_locator.find_executable("btor2code", subdir="build")
+        return tool_locator.find_executable("btor2c", subdir="build")
 
     def name(self):
-        return "BTOR2C"
+        return "Btor2C"
 
     def project_url(self):
         return "https://gitlab.com/sosy-lab/software/btor2c"
 
+    def version(self, executable):
+        return self._version_from_tool(executable, line_prefix="Btor2C version")
```

Comment:
The old version (when the binary was still `btor2code`) does not support `--version`.
The newer versions we used in TACAS papers support `--version`.
And ever since `--version` is introduced,
such prefix is printed.